### PR TITLE
Python wrapper script

### PR DESCRIPTION
**Pull request by emilmelnikov**

### CODE REVIEW

File: batch/models.py [hunk 1]
```diff
@@ -24,13 +24,17 @@ def get_queryset(self):
         return super().get_queryset().select_related("file")
 
 
+PROJECT_WORKFLOWS = ("Pixel Classification", "Object Classification")
```
Comment (m-novikov) [inline batch/models.py:27]: I think here snake_case should be used. While I know that in ilastik workflow name is used in this format and relied upon, I think can be easily misinterpreted as a human-readable string and changed. Related https://github.com/ilastik/ilastik/pull/2219

File: batch/ilastik_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+logging.Formatter.default_time_format = "%Y-%m-%dT%H:%M:%S"
+logging.Formatter.default_msec_format = "%s.%03dZ"
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    os.environ["PATH"] = os.pathsep.join(
+        [
+            os.path.expanduser("~/miniconda3/envs/ildev/bin"),
```
Comment (m-novikov) [inline batch/ilastik_wrapper.py:27]: Should this be configured somehow?

**Comment by emilmelnikov** [comment]
> Did you test it on HPC?

Yes, it worked at the time I've submitted this PR. However, I think it should be more configurable, and we also need to rebase this on top of rebased on top of https://github.com/ilastik/cloud_ilastik/pull/65, so it's probably better to scrap this altogether.

File: batch/ilastik_wrapper.py (hunk 2 above)
Comment (emilmelnikov) [reply batch/ilastik_wrapper.py:27]: This was copied verbatim from the existing script, but yes, I also don't like it.

Anyway, it's probably better not to copy scripts with the payload and config every time someone submits a job. We should embed this in a deployed environment somehow (which will probably include the whole Python + deps).

File: batch/models.py (hunk 1 above)
Comment (emilmelnikov) [reply batch/models.py:27]: See question/discussion in related.

**Comment by emilmelnikov** [comment]
Closed because this PR contains a lot of auth code, but HBP is migrating to FURMS (Fenix User and Resource Management Service).

**Pull request closed without merging**